resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/rest/kubernetes/KubernetesSparkRestServer.scala

```diff
@@ -275,5 +314,15 @@ private[spark] object KubernetesSparkRestServer {
     })
     barrier.await()
   }
+
+  private def fileToUtf8String(filePath: String, fileType: String) = {
```

Comment:
You can maybe replace this method with Guava's `Files.toString`

http://digitalsanctum.com/2012/11/30/how-to-read-file-contents-in-java-the-easy-way-with-guava/